doc/source/parameters.rst

```diff
@@ -0,0 +1,44 @@
+==============
+``/parameters``
+==============
+
+.. http:get:: /parameters
+
+    All the parameters contained in the tax and benefit system
```

Comment:
Ok, but I could mostly infer this with the name. What I'm looking for in the doc is the response format. I need to know that I will get an object whose keys will be parameters IDs and values objects with a single property, “description”, which is a String, and that I can use the ID to navigate to more information with the `parameter` endpoint.